h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -262,6 +264,29 @@ public boolean onExceptionalCompletion(Throwable ex, CountedCompleter caller) {
     }
 
     public abstract void computeImpl();
+    
+    public void computeEffectiveParameters() {
+      M model = _result.get();
+      if (model != null) {
+        model._effective_parms = (P) _parms.clone();
+        model.initEffectiveParam();
```

Comment:
I am not sure I like the computeEffectiveParameters/initEffectiveParam naming, I think both could be the same.

Reply:
renamed to be computeEffectiveParameters both